pvlib/test/test_atmosphere.py

```diff
@@ -113,3 +111,82 @@ def test_first_solar_spectral_correction_supplied():
 def test_first_solar_spectral_correction_ambiguous():
     with pytest.raises(TypeError):
         atmosphere.first_solar_spectral_correction(1, 1)
+
+
+def test_kasten96_lt():
+    """Test Linke turbidity factor calculated from AOD, Pwat and AM"""
+    timestamps = pd.DatetimeIndex(MELBOURNE_FL[0])
+    timestamps = timestamps.tz_localize('UTC').tz_convert('Etc/GMT+5')
```

Comment:
the ``conda list`` command in the travis builds reports pytz 2016.10 for both builds, so maybe it's pandas. I couldn't find anything about a change in the pandas behavior here, so maybe there's a bug in the newer version. strange.